model_cards/aliosm/ComVE-distilgpt2/README.md

```diff
@@ -0,0 +1,68 @@
+---
+language: "en"
+tags:
+- gpt2
+- exbert
+- commonsense
+- semeval2020
+- comve
+license: "mit"
+datasets:
+- https://github.com/wangcunxiang/SemEval2020-Task4-Commonsense-Validation-and-Explanation
+metrics:
+- bleu
+widget:
+- text: "Chicken can swim in water. <|continue|>"
+---
+
+# ComVE-distilgpt2
+
+## Model description
+
+Finetuned model on Commonsense Validation and Explanation (ComVE) dataset introduced in [SemEval2020 Task4](https://competitions.codalab.org/competitions/21080) using a causal language modeling (CLM) objective.
+The model is able to generate a reason why a given natural language statement is against commonsense.
+
+## Intended uses & limitations
+
+You can use the raw model for text generation to generate reasons why natural language statements are against commonsense.
+
+#### How to use
+
+You can use this model directly to generate reasons why the given statement is against commonsense using [`generate.sh`](https://github.com/AliOsm/SemEval2020-Task4-ComVE/tree/master/TaskC-Generation) script.
+
+*Note:* make sure that you are using version `2.4.1` of `transformers` package. Newer versions has some issue in text generation and the model repeats the last token generated again and again.
```

Comment:
We should be able to find why this is the case and fix the lib and/or the model. (cc @sshleifer)